pkg/cmd/roachtest/test_registry.go

```diff
@@ -137,7 +139,7 @@ func (r *testRegistryImpl) prepareSpec(spec *registry.TestSpec) error {
 	// may not be scheduled until a few hours in due to the CPU quota, individual
 	// tests should expect to take "less time". Longer-running tests require the
 	// weekly tag.
-	const maxTimeout = 18 * time.Hour
+	const maxTimeout = 7 * 24 * time.Hour
```

Comment:
Why this change?

Reply:
Oops - removed this change. thanks for catching this.